tools/Makefile

```diff
@@ -27,7 +27,7 @@ endif
 
 .PHONY: all clean
 
-all: kaldi.done python check_install
+all: kaldi.done python check_install extra
```

Comment:
It's better not to include `extra` to make the installation simple. What tools do you need to run?
Instead, you can add the installation check in `path.sh` see https://github.com/espnet/espnet/blob/d51251c799bd710e20804793e90c20f97cc6e60a/egs/csj/asr1/path.sh#L21-L33 as an example.